docs/docs/intro.mdx

```diff
@@ -13,7 +13,7 @@ geospatial charts.
 
 Here are a **few different ways you can get started with Superset**:
 
-- Download the [source from Apache Foundation's website](https://dist.apache.org/repos/dist/release/superset/1.0.0/)
+- Download the [source from Apache Foundation's website](https://dist.apache.org/repos/dist/release/superset/1.4.1/)
```

Comment:
I wonder if we should just link to https://dist.apache.org/repos/dist/release/superset/ instead? That way we don't have to keep remembering to update it each time / people can choose the version they want etc.

Reply:
I think if have a latest version should be a better way, for new downloads I don't know if oldest version would matter